orx/grantlee/0.2/OrxJs.qs

```diff
@@ -0,0 +1,280 @@
+
+    if (root.exporterProperties.includeComments) {
+        result += ' ; Source: ' + sprite.fullName;
+    }
+    
+    return result;
+}
+
+function tagIf(key, value, condition) {
+    if (condition) {
+        return tag(key, value);
+    }
+}
+
+function tag(key, value) {
+    var maxTagLength = 13; // I.e. TextureOrigin
+    while (key.length < maxTagLength) {
+        key += ' ';
+    }
+    return key + ' = ' + value;
+}
+
+function pointToString(point) {
+    return '(' + point.x + ', ' + point.y + ')';
+}
+
+function sortByName(arr) {
+    return arr.slice().sort(function(a, b) {
+        return a.trimmedName.localeCompare(b.trimmedName);
+    });
+}
+
+function pivotToString(sprite) {
+    var pivot = normPointToString(sprite.pivotPointNorm);
+    if (pivot === undefined) {
+        pivot = pointToString(sprite.pivotPoint);
+    }
+    return pivot;
+}
+
+function normPointToString(point) {
+    if (point.x == 0.5 && point.y == 0.5)
+        return 'center';
+    if (point.x == 0.0 && point.y == 0.0)
+        return 'top|left';
+    if (point.x == 1.0 && point.y == 1.0)
+        return 'bottom|right';
+    if (point.x == 1.0 && point.y == 0.0)
+        return 'top|right';
+    if (point.x == 0.0 && point.y == 1.0)
+        return 'bottom|left';
+    
+    // orx wants absolute coordinates!
+}
+
+function sizeToString(size) {
+    return '(' + size.width + ', ' + size.height + ')';
+}
+
+function printAnimation(root, sprite, texture, frameCount) {
+    if (!root.settings.autodetectAnimations || frameCount < 2) {
+        return;
+    }
+    
+    var animationSetid = getContainerId(sprite.trimmedName, 'animation-set');
+    var animId = getContainerId(sprite.trimmedName, 'anim');
+    
+    append(section(animationSetid),
+           tag('Texture', texture.fullName),
+           tagIf('KeepInCache', true, root.exporterProperties.keepInCache),
+           tag('FrameSize', sizeToString(sprite.untrimmedSize)),
+           tag('KeyDuration', root.exporterProperties.keyDuration),
+           tag('Digits', digitCount(frameCount)),
+           tag('StartAnim', animId),
+           tag(animId, frameCount));
+    
+    append();
+}
+
+function printFrame(root, sprite, id, parentId) {
+    append(frameSection(root, sprite, id, parentId),
+           tag('TextureOrigin', pointToString(sprite.frameRect)),
+           tag('TextureSize', sizeToString(sprite.frameRect)));
+
+    if (root.settings.writePivotPoints) {
+        append(tag('Pivot', pivotToString(sprite)));
+    }
+    append();
+}
+
+function pad(num, size) {
+    return ('000000000' + num).substr(-size);
+}
+
+function digitCount(num) {
+    return num.toString().length;
+}
+
+function getFrameInfo(root, sprite, texture, frameData) {
+    var name = sprite.trimmedName;
+    var parentId;
+    
+    if (root.settings.autodetectAnimations && frameData.frameCount > 1) {
+        var size = digitCount(frameData.frameCount);
+        var anim = 'anim' + pad(frameData.frameNo, size);
```

Comment:
Ah, I believe this might be linked to my previous comment (https://github.com/orx/texturepacker-exporter/pull/1/files#r515736586). The default mode is to use a variable-width suffixed counter, which shouldn't require padding.